### PR TITLE
Updates for documentation

### PR DESCRIPTION
**Pull request by martijn00**

### CODE REVIEW

**Comment by MarcBruins** [comment]
lots of improvements, looks good!

File: docs/_documentation/fundamentals/mvvmcross-overview.md [hunk 1]
```diff
@@ -250,44 +250,16 @@ public class SplashScreen : MvxSplashScreenActivity
...
 
+```c#
    using System;
    using System.Windows;
-   using Cirrious.CrossCore;
-   using Cirrious.MvvmCross.ViewModels;
-  `using Cirrious.MvvmCross.Wpf.Views;`
+   using MvvmCross.Core;
+   using MvvmCross.Core.ViewModels;
+  using MvvmCross.Wpf.Views;
```
Comment (MarcBruins) [inline docs/_documentation/fundamentals/mvvmcross-overview.md]: spacing is a bit off here

Comment (martijn00) [inline docs/_documentation/fundamentals/mvvmcross-overview.md]: Fixed now

File: docs/_documentation/fundamentals/mvvmcross-overview.md [hunk 2]
```diff
@@ -388,42 +361,19 @@ public class Setup : MvxAndroidSetup
...
-using MonoTouch.UIKit;
-using Cirrious.MvvmCross.Touch.Platform;
+using UIKit;
+using MvvmCross.iOS.Platform;
 
-namespace MyName.Touch
+namespace MyName.iOS
 {
-public class Setup : MvxTouchSetup
+public class Setup : MvxiOSSetup
```
Comment (nmilcoff) [inline docs/_documentation/fundamentals/mvvmcross-overview.md]: I think it is `MvxIosSetup`

**Pull request merged**